protocol.md

```diff
@@ -351,6 +351,20 @@ If the length of the upload exceeds the maximum, which MAY be specified using
 the `Tus-Max-Size` header, the Server MUST respond with the
 `413 Request Entity Too Large` status.
 
+The Client MAY include the entire or a chunk of the data, which is meant to be
+uploaded, in the body of the `POST` request. In this case, similar rules as for
```

Comment:
How about this wording?
The Client MAY include the entirety or a chunk of the data to be uploaded in the body of the `POST` request.